### PR TITLE
Removes mapping error bandaid

### PR DESCRIPTION
**Pull request by Cyberboss**
Don't be shit instead

### CODE REVIEW

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -41,12 +41,6 @@
 /obj/structure/table/proc/deconstruction_hints(mob/user)
 	to_chat(user, "<span class='notice'>The top is <b>screwed</b> on, but the main <b>bolts</b> are also visible.</span>")
 
-/obj/structure/table/Initialize()
-	. = ..()
-	for(var/obj/structure/table/T in src.loc)
```
Comment (ChangelingRain) [inline code/game/objects/structures/tables_racks.dm]: you'll need to update some lines below this, since they relied on this

**Comment by alexkar598** [comment]
Why remove it ?

**Comment by Cyberboss** [comment]
Because if a mapper puts 2 tables in one place it shouldn't be the code's job to fix it

File: code/game/objects/structures/tables_racks.dm (hunk 1 above)
Comment (Cyberboss) [reply code/game/objects/structures/tables_racks.dm]: done

**Comment by optimumtact** [comment]
except they keep doing it

**Pull request merged**